### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by woodybury** [proksh/atomize#5]
Hi, atomize. Fan of your sketch library and excited to see this react library. However, there's a lot of missing accessibility on the interactive components. For example, the drop-down is rendered as a plain div with no role, tab-index, or aria to let a keyboard dependent user interact with it or know the state. Likewise, the focus is not sent to the modal component. Those are just two examples. Is accessibility in the roadmap? I'm happy to help.

**Comment by ghost** [comment]
Wasn't in the roadmap, but I think I will have to work on it soon. I will appreciate any contribution or help 😀. Thank you.

**Comment by woodybury** [comment]
Great. Do you prefer a component to start with?

**Comment by proksh** [comment]
You can start with the smaller components like Anchor & Buttons. Or you can send me some good article on this so that I can learn and start working on it :)

**Comment by woodybury** [comment]
Ok, I'll take the first pass at anchor and button this week. I really enjoy Heydon Pickering's "Inclusive components" https://inclusive-components.design. The React docs do a fantastic job highlighting common patterns and offering more resources, dev tools, etc: https://reactjs.org/docs/accessibility.html

**Comment by proksh** [comment]
Thanks for your help 😃

**Comment by proksh** [comment]
I have resolved some issues with the update along with some dependency changes. Please check out.

I will continue to improve the keyboard accessibility. Thanks :)
